src/main/java/org/jabref/logic/exporter/OpenDocumentSpreadsheetCreator.java

```diff
@@ -46,22 +47,12 @@
      * Creates a new instance of OpenOfficeDocumentCreator
      */
     public OpenDocumentSpreadsheetCreator() {
-        super(Localization.lang("OpenDocument spreadsheet"), "ods", null, null, FileExtensions.ODS);
+        super(Localization.lang("OpenDocument spreadsheet"), "ods", null, null, FileType.ODS);
     }
 
-    @Override
-    public void performExport(final BibDatabaseContext databaseContext, final String file,
-            final Charset encoding, List<BibEntry> entries) throws IOException {
-        Objects.requireNonNull(databaseContext);
-        Objects.requireNonNull(entries);
-        if (!entries.isEmpty()) { // Only export if entries exists
-            OpenDocumentSpreadsheetCreator.exportOpenDocumentSpreadsheet(new File(file), databaseContext.getDatabase(), entries);
-        }
-    }
-
-    private static void storeOpenDocumentSpreadsheetFile(File file, InputStream source) throws IOException {
+    private static void storeOpenDocumentSpreadsheetFile(Path file, InputStream source) throws IOException {
 
-        try (ZipOutputStream out = new ZipOutputStream(new BufferedOutputStream(new FileOutputStream(file)))) {
+        try (ZipOutputStream out = new ZipOutputStream(new BufferedOutputStream(new FileOutputStream(file.toFile())))) {
```

Comment:
The last FileOutputStream could be replaced by the Files.newFileOutputStream which directly accepts a path

Reply:
Fixed.